src/OVAL/oval_component.c

```diff
@@ -1462,12 +1462,21 @@ static oval_syschar_collection_flag_t _oval_component_evaluate_VARREF(oval_argu_
 		return flag;
 	}
 
+	dIndent(1);
+	dI("Variable component references %s '%s'",
```

Comment:
It could be, because it's a very detailed information, but on the other hand, it's related to SCAP content, and not to the scanner implementation, if you are trying to figure out some problems with variables in your content, it might be beneficial the some of your OVAL_variables are referencing something else, so the evaluation flow would be more obvious.